src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -1017,6 +1017,154 @@ RppStatus rppt_slice_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** pixelate ********************/
+
+RppStatus rppt_pixelate_host(RppPtr_t srcPtr,
+                             RpptDescPtr srcDescPtr,
+                             RppPtr_t dstPtr,
+                             RpptDescPtr dstDescPtr,
+                             Rpp32s pixelationIntensity,
+                             RpptROIPtr roiTensorPtrSrc,
+                             RpptRoiType roiType,
+                             rppHandle_t rppHandle)
+{
+    RppLayoutParams srcLayoutParams = get_layout_params(srcDescPtr->layout, srcDescPtr->c);
+    RpptDescPtr interDescPtr = srcDescPtr;
+    RpptImagePatchPtr internalDstImgSizes = (RpptImagePatch *) calloc(dstDescPtr->n, sizeof(RpptImagePatch));
+    RpptROI *internalRoiTensorPtrSrc = (RpptROI *) calloc(dstDescPtr->n, sizeof(RpptROI));
+
+    for(int i = 0 ; i < dstDescPtr->n ; i++)
+    {
+        internalDstImgSizes[i].width = internalRoiTensorPtrSrc[i].xywhROI.roiWidth = roiTensorPtrSrc[i].xywhROI.roiWidth / pixelationIntensity;
```

Comment:
I feel this might not work well when input images of batch are of different sizes 
@r-abishek Please share your thoughts here

Reply:
@sampath1117 @HazarathKumarM 
If pixelationIntensity is a single number, but each roi is different for different images in the batch, then yes, the images may look very different in output. 
To fix that, think about taking a percentage pixelation from the user. But that percentage could be scaled internally for different image sizes in the batch, so that the net pixelated result looks same - like a 30% pixelation on 150x150 or a 30% pixelation on a 1024x768.

Reply:
It will be problem in above calculation if we go with batch size of values
@r-abishek @sampath1117 Please let me know on how to proceed in this issue

Reply:
we wont need batchsize of pixelintensity from outer api, we can calculate the pixelation intensity for each sample internally as per abisheks suggestion from the single value passed from user
```
To fix that, think about taking a percentage pixelation from the user. But that percentage could be scaled internally for different image sizes in the batch, so that the net pixelated result looks same - like a 30% pixelation on 150x150 or a 30% pixelation on a 1024x768.
```

You need to compute the max possible intermediate output size in batch and use that in the interDescPtr values
@r-abishek @snehaa8 
Please share your thoughts here

Reply:
Done